pkg/remote/remote_test.go

```diff
@@ -373,7 +373,7 @@ ARG OKTETO_GIT_COMMIT
 ARG OKTETO_GIT_BRANCH
 ARG OKTETO_INVALIDATE_CACHE
 
-RUN echo "$OKTETO_INVALIDATE_CACHE" > /var/lib/.oktetocachekey
+RUN mkdir -p /var/lib && echo "$OKTETO_INVALIDATE_CACHE" > /var/lib/.oktetocachekey
```

Comment:
`mkdir -p /var/lib` might fail, especially in not root images. I would avoid writing to `/var/lib` if possible, but if there is no better option, we should give a proper error if this command fails

Reply:
But that one is in the `$HOME`, which the user usually have access to it

Reply:
Since we already depend on $HOME, I would say `$HOME/oktetocachekey`